### PR TITLE
Fixes #10963 - partition table can't access os ids

### PR DESCRIPTION
**Pull request by mbacovsky**

### CODE REVIEW

File: app/models/ptable.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Ptable < Template
   scoped_search :on => :template, :complete_value => false, :rename => 'layout'
   scoped_search :on => :os_family, :rename => 'family', :complete_value => :true
 
-  attr_accessible :layout, :os_family
+  attr_accessible :layout, :os_family, :operatingsystem_ids
```
Comment (domcleal) [inline app/models/ptable.rb]: Also add operatingsystem_names, and probably also hostgroup + host ids/names?

**Comment by mbacovsky** [comment]
Updated @domcleal. I found out that these attrs are missing from the API docs for ptables controller. Should I add it as part of this PR?

**Comment by domcleal** [comment]
Up to you really, I'm happy if you want to add them.  I had hoped/expected for someone to come up with a way to auto-document them after the _ids/names functionality was added, but it didn't happen.

**Comment by mbacovsky** [comment]
Api docs for ptables V2 were added.

File: app/controllers/api/v2/ptables_controller.rb [hunk 2]
```diff
@@ -30,6 +30,9 @@ def show
           param :audit_comment, String, :allow_nil => true
           param :locked, :bool, :desc => N_("Whether or not the template is locked for editing")
           param :os_family, String, :required => false
+          param :operatingsystem_ids, Array, :desc => N_("Array of operating system IDs to associate with the partition table")
+          param :host_ids, Array, :desc => N_("Array of host IDs to associate with the partition table")
+          param :hostgroup_ids, Array, :desc => N_("Array of hostgroup IDs to associate with the partition table")
```
Comment (dLobatog) [inline app/controllers/api/v2/ptables_controller.rb:35]: I'll change it to 'host group' on merge

File: app/models/ptable.rb [hunk 3]
```diff
@@ -31,7 +31,8 @@ class Ptable < Template
   scoped_search :on => :template, :complete_value => false, :rename => 'layout'
   scoped_search :on => :os_family, :rename => 'family', :complete_value => :true
 
-  attr_accessible :layout, :os_family
+  attr_accessible :layout, :os_family, :operatingsystem_ids, :operatingsystem_names
+  attr_accessible :hostgroup_ids, :hostgroup_names, :host_ids, :host_names
```
Comment (dLobatog) [inline app/models/ptable.rb:35]: No need for the second `attr_accessible`, I'll remove on merge

**Comment by dLobatog** [comment]
Merged as 5fc2aed63fa0ab78432bbf91dbfd61c2543dfc97, thanks @mbacovsky !

**Pull request closed without merging**